metamer/application/pom.xml

```diff
@@ -144,6 +144,18 @@ FSF site: http://www.fsf.org. -->
             <artifactId>jboss-as-process-controller</artifactId>
             <version>7.2.0.Final</version>
             <scope>provided</scope>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.jboss.logging</groupId>
+                    <artifactId>jboss-logging</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+
+        <dependency>
+            <groupId>org.jboss.logging</groupId>
+            <artifactId>jboss-logging</artifactId>
+            <version>3.3.0.Final</version>
```

Comment:
Shouldn't this dependency be in scope "provided" when it's listed it manifest.mf?